ansible/roles/hoover/templates/initialize.d/hoover.sh

```diff
@@ -4,6 +4,17 @@ set -x
 
 cd /opt/hoover
 
+(
+    # create secret keys without echoing
+    set +x
+    echo "SECRET_KEY = '`openssl rand -base64 48`'" > /opt/hoover/search/hoover/site/settings/secret_key.py
```

Comment:
If the key already exists, does it get overwritten?

Reply:
Yeah, that's true. I'll make it not do that.